modules/get-started/pages/licensing/check-status/redpanda-operator.adoc

```diff
@@ -0,0 +1,103 @@
+= Check License Status and Feature Usage with Redpanda Operator
+:env-kubernetes: true
+:description: Learn how to monitor the status of an Enterprise Edition license and enterprise feature usage in Kubernetes using the Redpanda Operator.
+
+If your Redpanda cluster is deployed in Kubernetes using the Redpanda Operator, you can monitor the license status from the Redpanda custom resource. The operator integrates license monitoring into the resource's `status.conditions` and `status.license` fields, allowing you to get the license state with `kubectl`. These fields provide a detailed overview of your Redpanda license, including its status, expiration, and enterprise features currently in use.
+
+. Identify your Redpanda resources:
++
+[,bash]
+----
+kubectl get redpanda -A
+----
++
+Example output:
++
+[.no-copy]
+----
+NAMESPACE   NAME      LICENSE       READY   STATUS
+redpanda    cluster   Valid         True    Redpanda reconciliation succeeded
```

Comment:
if you just started the guide, the status will show like that " Redpanda reconciliation succeeded"?

Reply:
It's because the `redpanda` resource is for the cluster. So if the cluster is running, the status will be `reconciliation succeeded`.